packages/common/src/fixtures/xforms.ts

```diff
@@ -102,7 +102,7 @@ export class XFormResource<Type extends XFormResourceType> {
 				const service = new JRResourceService();
 				const parentPath = localPath.replace(/\/[^/]+$/, '');
 
-				service.activateFixtures(parentPath, ['file', 'file-csv']);
+				service.activateFixtures(parentPath, ['file', 'file-csv', 'images']);
```

Comment:
This is used to load files (forms and attachments) from the common package on the Preview Page when running the project in the local environment. I'm unsure yet what it needs to load attachments on the Preview Page when deployed in central.